lib/web/apiserver.go

```diff
@@ -2539,9 +2527,31 @@ func (h *Handler) clusterUnifiedResourcesGet(w http.ResponseWriter, r *http.Requ
 			}
 			unifiedResources = append(unifiedResources, server)
 		case types.DatabaseServer:
+			if !containsDb {
+				dbNames, dbUsers, err = getDatabaseUsersAndNames(accessChecker)
+				if err != nil {
+					return nil, trace.Wrap(err)
+				}
+				containsDb = true
+			}
 			db := ui.MakeDatabase(r.GetDatabase(), dbUsers, dbNames)
 			unifiedResources = append(unifiedResources, db)
 		case types.AppServer:
+			if !containsApp {
+				userGroups, err = apiclient.GetAllResources[types.UserGroup](request.Context(), clt, &proto.ListResourcesRequest{
+					ResourceType:     types.KindUserGroup,
+					Namespace:        apidefaults.Namespace,
+					UseSearchAsRoles: true,
+				})
+				if err != nil {
+					h.log.Debugf("Unable to fetch user groups while listing applications, unable to display associated user groups: %v", err)
```

Comment:
I'd make this info or warn since it results in user-facing behavior that may be surprising.

Reply:
I'll set to info to match the error received from RBAC check